libopenfpga/libpinconstrain/src/file_readers/pcf_reader.cpp

```diff
@@ -0,0 +1,27 @@
+#include "pcf_reader.h"
+
+bool PcfReader::read_pcf(const std::string &f)
+{
+    std::ifstream infile(f);
+    if (!infile.is_open())
+    {
+        cout << "ERROR: cound not open the file " << f << endl;
```

Comment:
Suggest to ``VTR_LOG`` for error messages.

Reply:
Fixed